builder/steps/activate.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 
+# Try personal activation
 if [[ -n "$UNITY_LICENSE" ]]; then
+  LICENSE_MODE="personal"
```

Comment:
I like this approach a lot! Could you place this variable under the comment though?